core/src/main/java/io/questdb/cairo/DefaultCairoConfiguration.java

```diff
@@ -891,6 +891,11 @@ public int getTxnScoreboardEntryCount() {
         return 8192;
     }
 
+    @Override
+    public String getType() {
```

Comment:
this looks misplaced in many ways:
- getType() is a cryptic name for a method (type of what)
- the variable names and method names do not match up, sometimes type, sometimes questdbType (what is questdb type?)
- if this flag is to trigger "auth" behaviour in the web console, it should just say that. There is no need to obfuscate the intent
- the auth impl in the server already has a switch, it can be disabled and it is technically disabled in OSS (not implemented). Why not use single switch?
- this property has nothing to do with CairoConfig, Cairo is storage. Web Console switches do not belong here

Reply:
Copied the description from enterprise PR into this one, it explains what i meant by QuestDB type, it is the type os the server: OSS or EE.
Agree on naming, `getType()` should be `getQuestDBType()` or maybe better to call it `getServerType()`.

It is not necessarily for auth, it is also for EE badge, and possibly other things in future.
Yes, the server type could be derived from the presence of ACL properties, but I thought that would be a hack, because this clear relationship could change in the future, so wanted to add a separate property which will not break anything if we decided to add ACL switch to OSS at some point.

Will refactor it, and move this property to a neutral place in the config, likely to top level ServerConfiguration, but that does not seem to have individual properties, apart from `isLineTcpEnabled()` which probably should not be there. :)

Reply:
sure, feel free to refactor other places if they are misplaced.

i am ok to to keep one switch for all Web Console ancillaries. Can we rename "questdbType" to "releaseType" ? or something like that, "questdb" is also a brand name, doesn't have a type.

Reply:
Using "releaseType" now, and refactored the code to get rid of `ServerConfiguration.isLineTcpEnabled()`.
Eventually I had to expose `acl.enabled` via the settings too, the release type on its own is not enough.
Ready for another review.